app/js/features/license/saga.js

```diff
@@ -22,10 +22,6 @@ import {eventChannel} from "redux-saga";
 export function *doBuyLicense() {
```

Comment:
The next stage would be to pass the price so we could use the normal doTransaction